### ISSUE TITLE
Question about GitHub for HW 1

### ISSUE THREAD

**Issue by mdscheuerell** [QERM514/assistance#7]
I would like your opinion(s) as to whether you would like to learn more about GitHub and its additional features that we are not using at the moment.

Please leave a comment below with your response.

**Comment by abigailkeller** [comment]
I'd like to learn more!

**Comment by ameliaduvall** [comment]
It would be helpful for me to learn more about GitHub too if we have the time to review other features.

**Comment by mjzahn** [comment]
I would also like to learn more!

**Comment by hmcmonagle** [comment]
Same!

**Comment by oxborrow** [comment]
Yes, please. That would be great.

**Comment by briellekthompson** [comment]
I'd like to learn more as well. I would love to talk about dealing with merging conflicts -I always struggle with that!

**Comment by jmlinds** [comment]
I'd be especially interested in learning how to use GitHub for version control if we have the time!

**Comment by Larnold1997** [comment]
I've used GitHub for version control, which I imagine is what we would cover (at least it is the first topic we would cover), so I would not personally be disappointed if we did not discuss version control.

I do not have much experience doing more advanced things on GitHub, such as merge conflicts, and would be interested in learning more about these topics.

**Comment by divyaOakGrovian** [comment]
Yes, I would like to learn additional features of GitHub, if time permits. I also wish to understand what advantages GitHub offers over other products such as Microsoft's Team Foundation Server. Thank you!

**Comment by greenamy808** [comment]
Being a newbie, I would like to learn more about GitHub and any other practical tips for projects moving forward in our careers.

**Comment by eileenbates** [comment]
learning Markdown is a higher priority for me, but if there is time I am definitely interested!

**Comment by jmh227** [comment]
I'd like to learn more too!

**Comment by don-radcliffe** [comment]
I'd definitely like to learn more!

**Comment by johnsr26** [comment]
I wouldn't mind spending a little time on github. I've learned *how* to use git and github for version control, but haven't yet figured out the best way to incorporate into a workflow. It'd be helpful for me to see it "in action" and maybe spend some time talking about how to set up projects with version control in mind.

**Comment by clohrn** [comment]
I would like to learn more about how to use GitHub for projects and how to interact with information on GitHub. I know very little about the world of GitHub and have always been a bit daunted!

**Comment by RobTour** [comment]
Yes, please!  That would be great.

**Comment by kpjohnson** [comment]
I think rmarkdown is a priority for me as well. I think I would like to become GitHub literate (i.e., know how to navigate around a repo, dealing with issues, and perhaps hearing about how you set up a repo for a new research project.

**Comment by lee-qi** [comment]
I don't think I take enough advantage of Github and its features, so it would be great to spend some time learning this!

**Comment by aeluro** [comment]
I'm glad to be learning rmarkdown, which is more of a priority for me than Github, although I am always interested in making use of free resources.

**Comment by krmistry** [comment]
I would appreciate learning more about Github, but I also think I could find that in other places, so the other learning objectives of this class are a higher priority for me than this (but if there's extra time, then that would be great!)

**Comment by sarahtanja** [comment]
Yes, I'd be interested in using Github more frequently... I've hosted a project on Github after it was complete, but would like to move into using it daily for version control (and as a backup!)

**Comment by fishnick62** [comment]
I am interested in learning more about Github. It seems like it is widely used and useful, and I know nothing about it!
